widgets/controls.pas

```diff
@@ -1467,7 +1481,18 @@ function TControl.HandleClick(AEvent: TJSMouseEvent): boolean;
 begin
   AEvent.StopPropagation;
   Click();
-  Result := True;
+  Result := true;
+end;
+
+function TControl.HandleContextMenu(AEvent: TJSWheelEvent): boolean;
+begin
+  if Assigned(PopupMenu) then
+  begin
+    AEvent.preventDefault;
+    writeln('Context');
+    PopupMenu.Popup(Round(AEvent.clientX), Round(AEvent.clientY));
```

Comment:
The numbers in ```clientX``` and ```clientY``` _should_ be whole numbers, so wouldn't ```Trunc``` be faster than ```Round```?

Reply:
Somehow I didn't think about it. Thanks.